examples/microGPT.py

```diff
@@ -71,10 +71,12 @@ def __init__(
                     },
                 },
                 "feedforward_config": {
-                    "name": "FusedMLP",  # Use MLP if Triton is not available
+                    "name": "MixtureOfExperts",  # Use MLP if Triton is not available
```

Comment:
pulling in MoE becomes as simple as that (though distributed training adds another layer of complication)